src/main/java/io/jenkins/plugins/orka/client/OrkaVMConfig.java

```diff
@@ -31,6 +40,10 @@ public String getBaseImage() {
         return this.baseImage;
     }
 
+    public int getMemory() {
```

Comment:
Seems you'll need to edit the equals and hashCode methods.
Check all other types as well. They may need edit, too